kornia/color/colormap.py

```diff
@@ -173,20 +199,27 @@ def apply_colormap(input_tensor: Tensor, colormap: ColorMap) -> Tensor:
 
 
 class ApplyColorMap(Module):
-    r"""Module that apply to a gray tensor (integer tensor) a colormap.
+    r"""Class for applying a colormap to images.
 
     Args:
```

Comment:
can you add a sample image here? need to add an entry in `docs/generate_examples.py` -- can be something like the example in the description of your PR